grunttasks/json2html.js

```diff
@@ -0,0 +1,50 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var path = require('path');
+
+// Four steps are performed to lint the files:
+//  1. Clean the directories
+//  2. Generate the JSON files from the PO files
+//  3. The JSON files are then written out to temporary HTML files
+//  4. The HTML files are linted
+
+module.exports = function (grunt) {
+  'use strict';
+  grunt.registerMultiTask('l10n-json-to-html', 'Convert l10n JSON files to HTML', function () {
+    var files = 0;
+    this.files.forEach(function (file) {
+      var HTMLcontent = '';
+      var src = file.src[0];
+      var pathname = src.split(path.sep);
+      var localeStrings = grunt.file.readJSON(src);
+      for (var val in localeStrings) {
+        var value = localeStrings[val];
+        if (typeof value !== 'object' && value !== '' && isNaN(value)) {
+          HTMLcontent += value.toString() + '\n\n';
+        }
+      }
+      if (HTMLcontent !== '') {
+        var tmpDir = '.tmp';
```

Comment:
I see `.tmp` sprinked around, that should be moved into the configuration task I mentioned earlier.